Formula/apng2gif.rb

```diff
@@ -0,0 +1,20 @@
+class Apng2gif < Formula
+  desc "Convert APNG animations into animated GIF format."
+  homepage "https://apng2gif.sourceforge.io/"
+  url "https://downloads.sourceforge.net/apng2gif/apng2gif-1.8-src.zip"
+  sha256 "9a07e386017dc696573cd7bc7b46b2575c06da0bc68c3c4f1c24a4b39cdedd4d"
+
+  depends_on "libpng"
+
+  def install
+    inreplace "Makefile", "gcc", ENV.cc
+    inreplace "Makefile", "-s", ""
```

Comment:
ld(1) Manual say:

```
-s Completely strip the output, including removing the symbol table. This file format variant is no longer supported. This option is obsolete
```
And [test failed](https://jenkins.brew.sh/job/Homebrew%20Core%20Pull%20Requests/2998/version=yosemite/testReport/brew-test-bot/yosemite/install_apng2gif/) when with `-s` (only in yosemite)

```
==> make
gcc -Wall -pedantic -O2 -o apng2gif apng2gif.cpp -s -lstdc++ -lm -lpng -lz
ld: internal error: atom not found in symbolIndex(__ZN5Image4freeEv) for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make: *** [all] Error 1
```

So I thought that I should remove it, and [test passed](https://jenkins.brew.sh/job/Homebrew%20Core%20Pull%20Requests/3000/version=yosemite/testReport/brew-test-bot/yosemite/install_apng2gif/) after that.

```
==> make
clang -Wall -pedantic -O2 -o apng2gif apng2gif.cpp  -lstdc++ -lm -lpng -lz
==> Cleaning
```

Did I miss something?